tests/integration/test_case.go

```diff
@@ -779,7 +779,26 @@ type BackupImport struct {
 	ExpectedError string
 }
 
-// CheckNodesIdentities is an action that will check the identities of the nodes.
-// It asserts that the `GetNodeIdentity` method returns an identity of the node that matches
-// the first generated identity (with index 0 in the nodes array of identities) for the node.
-type CheckNodesIdentities struct{}
+// GetNodeIdentity is an action that calls the [DB.GetNodeIdentity] method and asserts the result.
+// It checks if a node at the given index has an identity matching another identity under the same index.
+type GetNodeIdentity struct {
+	// NodeID holds the ID (index) of a node to get the identity from.
+	NodeID int
+
+	// ExpectedIdentityName is the name associated with the expected identity of the node.
```

Comment:
suggestion: Perhaps worsened by the previous missing documentation for this attribute, but can you describe what `IdentityName` is? I am specially wondering about the `name` part as opposed to what we have before in tests as `identity Index`